Data/core_rulebook/cr_abilities.lua

```diff
@@ -7113,12 +7155,15 @@ DefineAbility({
   AllowMultiple=true,
   Stackable=true,
   Cost=0,
-  Chooses={
-    ChooseAbilityselection(function (character, ability)
-      return (ability.Category == "FEAT") or (true)
+  Choice={
+    Choose=ChooseAbilitySelection(function (character, ability)
```

Comment:
The name of this ability and the Choice seem to be in conflict.  The filter seems to be anything _except_ feats, but it's called "Add a Feat"

Reply:
This filter is "if its Category is not FEAT return false, otherwise return true"